### ISSUE TITLE
Call to undefined method on ReportController

### ISSUE THREAD

**Issue by Partyschaum** [Jimdo/reportbook#54]
If you have a look at the log output from the development server you'll see the following error message:

```
PHP Fatal error:  Uncaught Error: Call to undefined method Jimdo\Reports\Web\Controller\ReportController::jsAction() in /Users/hauke/code/github.com/jimdo/reportbook/src/Web/Router.php:72
Stack trace:
#0 /Users/hauke/code/github.com/jimdo/reportbook/app/router.php(15): Jimdo\Reports\Web\Router->dispatch('/report/js/boot...')
#1 {main}
  thrown in /Users/hauke/code/github.com/jimdo/reportbook/src/Web/Router.php on line 72
```

Please investigate. Find the cause and fix it. 👍

  - [x] Implement `ActionNotFoundException` and throw when appropriate
  - [x] Make routing configurable: add paths which should pass through to filesystem
    - e.g. `$router->ignorePath('/css/')` ~~should then lead to the router serving everything under `/css/*` directly from disk~~
  - [x] Return `400` response code if controller or action was not found

**Comment by Partyschaum** [comment]
Think about possible solutions. I think there are more than one.

**Comment by SharktheFire** [comment]
Relates to #46

**Comment by SharktheFire** [comment]
Everything that contains a dot should get ignored by the router.

**Comment by SharktheFire** [comment]
Everything that contains a specific type should get ignored by the router.

**Comment by SharktheFire** [comment]
You can have a list with filenames which the router ignores.

**Comment by SharktheFire** [comment]
You can predefine the actions which the controller uses.

**Comment by SharktheFire** [comment]
Implemented in #58
